mozilla/foundation.mozilla.org
# Change donate modal default 'no' button text

Closes #4880 

To test, 

1. log into review app's CMS
2. From sidebar, go to Snippet -> Donation CTA -> Add Donation Modal
3. verify that the default text for Dismiss text is `No thanks`
